Exercise 2/Start/BackgroundDownload/DownloadController.cs

```diff
@@ -69,7 +69,7 @@ public override void ViewDidLoad()
 				// Cancel all pending downloads.
 				if(this.session != null)
```

Comment:
Another diff between Start and Completed, though unrelated to your changes. (I'm just running a comparison to make sure the only changes made are exercise steps.) The Completed solution doesn't check for a non-null `session` here for some reason. I'll make them match up since it doesn't seem to be anything we instruct someone to edit.